config/locales/cy/brexit_landing_page.yml

```diff
@@ -1,60 +1,49 @@
 cy:
   brexit_landing_page:
     meta_title: "Brexit"
-    meta_description: "Gwybodaeth a chanllawiau Brexit ar sut i baratoi ar gyfer Brexit heb fargen."
-    page_title: "Brexit"
-    page_intro: |
-      <p>Mae bargen Brexit wedi ei gytuno mewn egwyddor gyda’r UE.</p>
-      <p>Bydd angen i’r DU a’r UE gymeradwyo ac arwyddo’r cytundeb ymadael. Byddent wedyn yn dechrau trafod trefniadau newydd. Bydd cyfnod pontio i baratoi ar gyfer y rheolau newydd.</p>
-      <p>Gall y DU adael heb fargen os nad yw’r cytundeb ymadael wedi ei gymeradwyo erbyn 31 Ionawr 2020, neu ar ddiwedd y cyfnod pontio.</p>
-      <p>Darganfyddwch beth ddylech chi, eich teulu neu’ch busnes wneud os yw’r DU yn gadael yr UE heb fargen.</p>
-      <p><a href="/get-ready-brexit-check">Gwiriwch sut i baratoi ar gyfer Brexit heb fargen</a></p>
-    chevron_banner: "Gwiriwch beth sydd angen i chi wneud os nad oes cytundeb"
-    finder_heading: "Pori drwy’r holl wybodaeth sy’n gysylltiedig â Brexit:"
-    call_to_action_intro: "Atebwch ychydig o gwestiynau i ddarganfod sut y dylech chi neu’ch busnes baratoi."
-    all_brexit_information: "Holl wybodaeth Brexit"
-    email_intro: "I gael y manylion diweddaraf"
-    email: "Cofrestrwch i gael hysbysiadau e-bost am Brexit"
-    browse: "Pori’r canllawiau Brexit"
-    share_links: "Rhannu’r dudalen hon"
-    sections:
-      services: Gwasanaethau
-      guidance_and_regulation: Canllawiau a rheoliadau
-      news_and_communications: Newyddion a chyfathrebu
-      research_and_statistics: Ymchwil ag ystadegau
-      policy_and_engagement: Polisi ac ymgynghoriadau
-      transparency: Tryloywder a datganiadau rhyddid gwybodaeth
+    meta_description: "Mae’r DU a’r UE wedi cytuno ar gytundeb ymadael. Dysgwch beth fydd yn digwydd nesaf."
```

Comment:
re-ordered a lot of these to match the page, tried to remove ones i couldn't see being used anymore and renamed a bunch to match content designers jargon